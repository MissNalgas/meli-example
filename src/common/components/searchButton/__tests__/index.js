import renderer from 'react-test-renderer';
import SearchButton from '../';

test('SearchButton renders correctly', async () => {
	const tree = renderer.create(<SearchButton/>).toJSON();
	expect(tree).toMatchSnapshot();
});
