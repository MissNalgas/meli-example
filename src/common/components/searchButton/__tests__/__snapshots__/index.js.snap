// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton renders correctly 1`] = `
<button
  className="bg-light-gray h-10 w-10 grid place-content-center rounded"
>
  <img
    alt="Search icon"
    data-nimg="1"
    decoding="async"
    height={22}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fic_Search%402x.png&w=48&q=75"
    srcSet="/_next/image?url=%2Fic_Search%402x.png&w=32&q=75 1x, /_next/image?url=%2Fic_Search%402x.png&w=48&q=75 2x"
    style={
      {
        "color": "transparent",
      }
    }
    width={22}
  />
</button>
`;
